pkg/process/config/config.go

```diff
@@ -437,12 +414,6 @@ func loadEnvVariables() {
 		{"DD_BIND_HOST", "bind_host"},
 		{"HTTPS_PROXY", "proxy.https"},
 		{"DD_PROXY_HTTPS", "proxy.https"},
```

Comment:
Could you double check if this list is complete in terms of special case handling of env vars - if they are already handled by `pkg/config` let's remove them from here. I believe you are already handling `process_config.process_discovery.enabled`

Reply:
Just fixed this! Thank you for letting me know, I didn't realize that I had missed a few